parquet/src/test/java/org/apache/iceberg/parquet/TestParquetSchemaUtil.java

```diff
@@ -464,4 +588,103 @@ private Type map(Integer id, String name, Repetition repetition, Type keyType, T
     }
     return builder.named(name);
   }
+
+  @Test
+  public void testPruneColumns1() {
```

Comment:
Test names need to be cleaned up here as well. Can we have a more descriptive names?